bot/seasons/evergreen/bookmark.py

```diff
@@ -45,18 +49,48 @@ def __init__(self, bot: commands.Bot):
         embed.set_author(name=target_message.author, icon_url=target_message.author.avatar_url)
         embed.set_thumbnail(url=bookmark_icon_url)
 
+        error_embed = discord.Embed(
+            title=random.choice(ERROR_REPLIES),
+            description=f"Please enable your DMs to receive the bookmark.",
+            colour=Colours.soft_red
+        )
+
         try:
             await ctx.author.send(embed=embed)
         except discord.Forbidden:
-            error_embed = discord.Embed(
-                title=random.choice(ERROR_REPLIES),
-                description=f"{ctx.author.mention}, please enable your DMs to receive the bookmark",
-                colour=Colours.soft_red
-            )
             await ctx.send(embed=error_embed)
-        else:
-            log.info(f"{ctx.author} bookmarked {target_message.jump_url} with title '{title}'")
-            await ctx.message.add_reaction(Emojis.envelope)
+
+        log.info(f"{ctx.author} bookmarked {target_message.jump_url} with title '{title}'.")
+        await ctx.message.add_reaction(Emojis.envelope)
+        await ctx.message.add_reaction(Emojis.pin)
+
+        embed.add_field(
+            name=f'Bookmarked from {ctx.author.name}.',
+            value=f'[Visit original message]({ctx.message.jump_url})',
+            inline=False
+        )
+
+        def check(reaction: discord.Reaction, user: discord.User) -> bool:
+            return (
+                user != self.bot.user
+                and reaction.emoji == Emojis.pin
+                and reaction.message == ctx.message
+                and user not in sent_person
+            )
+
+        sent_person = {ctx.author}  # set of id who got the message
+        while True:
+            reaction, user = await self.bot.wait_for("reaction_add", timeout=10.0, check=check)
+            log.info(f"{user.name} bookmarked {target_message.jump_url} with title '{title}' from '{ctx.author}'.")
+            try:
+                await user.send(embed=embed)

```

Comment:
`wait_for` is the one that will raise `asyncio.TimeoutError` and not `await user.send(embed=embed)` while `user.send()` will raise `discord.Forbidden` - so timeout will effectively raise an error.